lago/utils.py

```diff
@@ -533,12 +534,13 @@ def load_virt_stream(virt_fd):
         dict: Loaded virt config
     """
     try:
-        virt_conf = json.load(virt_fd)
+        virt_conf = collections.OrderedDict()
+        virt_conf = json.load(virt_fd, object_pairs_hook=collections.OrderedDict)
     except ValueError:
         virt_fd.seek(0)
         virt_conf = yaml.load(virt_fd)
```

Comment:
Is there any way to make sure the order is also preserved when loading yaml files? As well, it's the format we are moving everything to